bigquery/src/test/java/com/google/cloud/hadoop/io/bigquery/BigQueryConfigurationTest.java

```diff
@@ -38,7 +38,7 @@
 
   @SuppressWarnings("DoubleBraceInitialization")
   private static final Map<String, Object> expectedDefaultConfiguration =
-      new HashMap<String, Object>() {
```

Comment:
Are these changes related to Java 11 ?

Reply:
Yes, these are Java 11+ clean ups that are not applicable to Java 8.